dotCMS/src/integration-test/java/com/dotmarketing/startup/StartupTasksExecutorTest.java

```diff
@@ -0,0 +1,54 @@
+package com.dotmarketing.startup;
+
+import static org.junit.Assert.assertEquals;
+
+import com.dotcms.util.IntegrationTestInitService;
+import com.tngtech.java.junit.dataprovider.DataProvider;
+import com.tngtech.java.junit.dataprovider.DataProviderRunner;
+import com.tngtech.java.junit.dataprovider.UseDataProvider;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+@RunWith(DataProviderRunner.class)
+public class StartupTasksExecutorTest {
+
+    @DataProvider
```

Comment:
Codacy found an issue: [JUnit 4 tests that execute tests should use the @Test annotation, JUnit 5 tests should use @Test, @RepeatedTest, @TestFactory, @TestTemplate or @ParameterizedTest](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6354433)